packages/client/src/components/groupSettings/__tests__/members.spec.tsx

```diff
@@ -20,15 +48,14 @@ function setup() {
 }
 
 describe('GroupSettingsMembers Component', () => {
-  it('Should render group settings members', () => {
+  it('Should render group settings members', async () => {
```

Comment:
According to [this](https://github.com/InfuseAI/primehub-console/pull/253/files#diff-87de329c876b9b2445a6274b597b37243894bbea3909a1b44b6d2aa06c69de3eR18-R20), did you mind adding a test case for loading status?

```tsx
it('should render group setting members with loading status', () => {
  const { group } = setup();
  render(<GroupSettingsMembers group={group} />, {
    wrapper: AllTheProviders,
  });
  expect(screen.getByText('loading...'));
});
```

Reply:
Sure thing. Thanks